chain/indexer-primitives/src/lib.rs

```diff
@@ -42,4 +41,5 @@ pub struct IndexerShard {
     pub shard_id: types::ShardId,
     pub chunk: Option<IndexerChunkView>,
     pub receipt_execution_outcomes: Vec<IndexerExecutionOutcomeWithReceipt>,
+    pub state_changes: views::StateChangesView,
```

Comment:
do we have any tests for this ?

Reply:
We had a conversation re: indexer and tests with @matklad recently

Here's the start https://github.com/near/nearcore/pull/6195#issuecomment-1023053576

Reply:
so - no tests ? :-(